orttraining/orttraining/python/training/ortmodule/ortmodule.py

```diff
@@ -102,6 +102,8 @@ def __init__(self, module, debug_options=None):
         # Assign self._is_initialized to True after all the ORTModule class attributes have been assigned
         # else, they will be assigned to self._torch_module.original_module instead.
         self._is_initialized = True
+        if not module.training:
+            self.eval()
```

Comment:
let's try to keep, `self._is_initialized = True` as the last step on the constructor.

Reply:
done